test/e2e/framework/network/utils.go

```diff
@@ -673,6 +673,10 @@ func (config *NetworkingTestConfig) setup(selector map[string]string) {
 	} else {
 		config.NodeIP = e2enode.FirstAddress(nodeList, v1.NodeInternalIP)
 	}
+
+	ginkgo.By("Waiting for service to expose endpoint")
```

Comment:
While this change seems fine to me, I'm not sure I understand how this is related the the contrack issue - where is this function used there?

Reply:
is not related to the conntrack issue per se, is related to one of the comments you did in that issue

> when creating the first pod, you wait for corresponding endpoint for that being created;

I've noticed that the tests that are using these methods are failing sporadically because they are not able to get the endpoints, and I noticed that the test was not waiting for the endpoint objects to be created.